ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/flow-trackings.j2

```diff
@@ -25,7 +25,7 @@ Sample: {{ flow_tracking.sample | arista.avd.default("default") }}
 {%                             do applied_on.append(ethernet_interface.name) %}
 {%                         endif %}
 {%                     endfor %}
-{%                     for port_channel_interface in port_channel_interfaces %}
+{%                     for port_channel_interface in port_channel_interfaces | arista.avd.default([]) %}
```

Comment:
Please do the same for ethernet_interfaces.

Reply:
Done in commit a4e4206